app/scripts/lib/experiment.js

```diff
@@ -17,7 +17,6 @@ define(function (require, exports, module) {
    * Experiments that are created on startup in `chooseExperiments`.
    */
   const STARTUP_EXPERIMENTS = {
-    'disabledButtonState': BaseExperiment,
```

Comment:
nit: Remove from [docs](https://github.com/mozilla/fxa-content-server/blob/0b3269ed5f4c4a833e6e9d66793389904db5a24b/docs/query-params.md#options-15)

Reply:
> nit: Remove from docs

Thanks for catching that!

Reply:
I was just looking `control` is still available for `emailFirst`. Maybe that section needs to be cleaned up.

Reply:
I'm getting my PRs confused. yes, I'll remove `disabledButtonState`.